src/routes/safe/components/Balances/SendModal/screens/SendFunds/index.tsx

```diff
@@ -75,6 +75,10 @@ type SendFundsProps = {
   amount?: string
 }
 
+const InputAdornmentChildSymbol = ({ symbol }: { symbol?: string }): ReactElement => {
+  return <>{symbol}</>
+}
```

Comment:
why is it needed?

Reply:
![image](https://user-images.githubusercontent.com/21086218/105891244-29521600-5fef-11eb-8906-60a0d109a855.png)